### ISSUE TITLE
South Park: Chef's Luv Shack does not start

### ISSUE THREAD

**Issue by carnage702** [xjsxjs197/WiiSXRX_2022#28]
Both intrepreter and dynarec just give black screen SLUS-00997

**Comment by xjsxjs197** [comment]
I think there is something wrong with the counter code. The programs in the MainCounter branch can run this game. Some games have a good speed increase effect, while others have a low frame rate

**Comment by xjsxjs197** [comment]
Both main branch and mainCounter brahch can work normally.

**Comment by carnage702** [comment]
yeah i tested it works great now so when its the next official release to change the name to wiistation? hopefully soon xD
